calculator.js

```diff
@@ -1,73 +1,128 @@
 const PLUS = '+'; 
 const SUBTRACT = '-';
-const DELETE = "delete";
+const MULTIPLY = '*';
+const DIVIDE = '/';
+const DELETE = "⌂";
 const EQUAL = '=';
-const DISPLAYzERO = `0`;
-const EMPTYsTRING = "";
-const EMPTYcHARACTER = '';
+const DISPLAY_ZERO = `0`;
 const BASE = 10;
-const OUTCOME = document.querySelectorAll("input");
-/*
- * I don´t know if the behavior of this object is like a struct:
- * Questions: 
- *  It has to be declared first as variable before use it or not?
- *  Also I dont't know if I can set the variables that are on it.
- *  Because of it's working, I'm not sure.
+const CLEAR =  'C'
+const NUMBERS = document.querySelectorAll(".numbers");
+const ACTIONS = document.querySelectorAll(".actions");
+const OPERATORS = document.querySelectorAll(".operators");
```

Comment:
`NUMBERS, ACTIONS, and OPERATORS` are not constants. They depend on the `document` and the `selector. For example name them as `const numberElements = document.querySelectorAll(".numbers");`.